pkg/config/config.go

```diff
@@ -143,22 +147,31 @@ type AudioConfig struct {
 	SmoothIntervals uint32 `yaml:"smooth_intervals"`
 }
 
+type StreamTrackerPacketConfig struct {
+	SamplesRequired uint32        `yaml:"samples_required"` // number of samples needed per cycle
+	CyclesRequired  uint32        `yaml:"cycles_required"`  // number of cycles needed to be active
+	CycleDuration   time.Duration `yaml:"cycle_duration"`
+}
+
+type StreamTrackerFrameConfig struct {
+	MinFPS float64 `yaml:"min_fps"`
+}
+
 type StreamTrackerConfig struct {
-	SamplesRequired       uint32        `yaml:"samples_required"`
-	CyclesRequired        uint32        `yaml:"cycles_required"`
-	CycleDuration         time.Duration `yaml:"cycle_duration"`
-	BitrateReportInterval time.Duration `yaml:"bitrate_report_interval"`
+	BitrateReportInterval map[int32]time.Duration             `yaml:"bitrate_report_interval,omitempty"`
+	ExemptedLayers        []int32                             `yaml:"exempted_layers,omitempty"`
+	PacketTracker         map[int32]StreamTrackerPacketConfig `yaml:"packet_tracker,omitempty"`
+	FrameTracker          map[int32]StreamTrackerFrameConfig  `yaml:"frame_tracker,omitempty"`
```

Comment:
Reworking this to contain all the stream tracker config for a specific track source. Top level config then has only `Video` and `Screenshare`.